consensus/module_consensus_state_sync.go

```diff
@@ -39,3 +63,56 @@ func (m *consensusModule) metadataSyncLoop() {
 	// requests metadata from peers
 	// sends received metadata to the metadataReceived channel
 }
+
+func (m *consensusModule) maxPersistedBlockHeight() (uint64, error) {
+	readCtx, err := m.GetBus().GetPersistenceModule().NewReadContext(int64(m.CurrentHeight()))
+	if err != nil {
+		return 0, err
+	}
+	defer readCtx.Release()
+
+	maxHeight, err := readCtx.GetMaximumBlockHeight()
+	if err != nil {
+		return 0, err
+	}
+
+	return maxHeight, nil
+}
+
+func (m *consensusModule) verifyBlock(block *coreTypes.Block) error {
+	return nil
+}
```

Comment:
s/verify/validate

Can you add comments or TODOs on what kind of validation you want to do here?